.mvn/excludes.txt

```diff
@@ -1127,9 +1116,6 @@
 :camel-quarkus-xj-deployment
 :camel-quarkus-xj-integration-test
 :camel-quarkus-xj-parent
-:camel-quarkus-xml-io-dsl
```

Comment:
Is the XML DSL definately required? Neither CQ or Camel-K product support it AFAIK.

Reply:
I included it based on @lburgazzoli 's statement

> camel-quarkus-yaml-dsl, camel-quarkus-java-joor-dsl, camel-quarkus-xml-io-dsl  are the support extension for the languages we support in camel-k

Reply:
I can remove it again if there is a strong enough evidence that XML is not supported in Camel K

Reply:
Docs only mention Java & YAML:

https://access.redhat.com/documentation/en-us/red_hat_integration/2021.q3/html/getting_started_with_camel_k/developing-and-running-camel-k-integrations

Reply:
Yes, I think it was my mistake because as a long time camel rider, I assumed XML and Java were supported by default.

Reply:
OK, let's exclude XML